src/App.js

```diff
@@ -68,11 +120,19 @@ export default class App {
     return pick(this.config, keys);
   }
 
+  static async override(defaultHandler, overrideHandler) {
+    if (overrideHandler !== undefined) {
+      await overrideHandler(this);
```

Comment:
Just a note that in the docs we should be clear that override handlers must return promises

Reply:
Technically they don't have to.  `await` will do the right thing, which is nice. 

From here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await

"If the value of the expression following the await operator is not a Promise, it's converted to a resolved Promise."